WIP - Enterprise 4.12: Credential DPA cherry pick

anarnold97 opened the pull request:
### Cherry pick for 4.12

Cherry Picked from 0b495fbf289ee49b3fd5aa190ef4c5423cdcbd43 xref: https://github.com/openshift/openshift-docs/pull/78934

### Jira

* [OADP-1443](https://issues.redhat.com/browse/OADP-1443)

### Version(s):

* OCP 4.12

Issue:
<!--- Add a link to the Bugzilla, Jira, or GitHub issue, if applicable. --->

Link to docs preview:
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->

QE review:
- [ ] QE has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

Additional information:
<!--- Optional: Include additional context or expand the description here.--->

<!--- After you open your PR, ask for review from the OpenShift docs team:
  For community authors: Tag @openshift/team-documentation in a GitHub comment.--->

anarnold97 commented:
/label oadp

anarnold97 commented:
@prasad - please can I ask you to do a quick double-check for this OCP 4.12 cherry pick for @shdeshpa07

I am pretty sure it is correct but given the different file names, it would be great if you could just quickly check

Thanks

PrasadJoshi12 commented on modules/oadp-installing-dpa.adoc:
```diff
@@ -169,39 +181,39 @@ spec:
+          bucket: <bucket_name> # <7>
+          prefix: <prefix> # <8>
+  snapshotLocations: # <9>
+          snapshotLocation: us-west1 # <10>
```
```suggestion
          region: us-west1 # <10>
```

PrasadJoshi12 replied:
snapshotLocation field is added twice, can you fix it ?

PrasadJoshi12 commented on modules/oadp-installing-dpa.adoc:
```diff
@@ -113,35 +121,38 @@ spec:
           resourceGroup: <azure_resource_group>
           subscriptionId: <azure_subscription_id>
           incremental: "true"
         name: default
```
VSL doesn't support specifying name

anarnold97 commented:
@PrasadJoshi12 - I have updated this and removed the old module and pointed to the `OADP 1.2 and earlier DPA`

But looking at [OpenShift Operator Life Cycles](https://access.redhat.com/support/policy/updates/openshift_operators)

![image](https://github.com/user-attachments/assets/88e517fe-502b-4769-95bc-80c179b9d91b)

Please can I ask if we should we add the 1.3 DPA to the assembly?
